input/includes/menu.xml

```diff
@@ -16,6 +16,9 @@
       <b class="caret"></b>
     </a>
     <ul class="dropdown-menu">
+      <li>
```

Comment:
Commenting here to get an inline comment. 
Update 1.3 How to Read this Guide to include Artefacts Summary page description.

Reply:
Artefacts Summary page description was present, created grouping page similar to 'guidance.html', shifted 'Artefacts Summary' to be child of this page.

Reply:
Thanks Jaymee, confirmed.